client/css/style.css

```diff
@@ -573,6 +573,7 @@ button {
 }
 #chat .wrap,
 #chat .text a {
+	cursor: pointer;
```

Comment:
Is it really necessary to add a cursor pointer to `a` links as opposed to default behavior?